third_party/perfetto/METADATA

```diff
@@ -1,3 +1,22 @@
+name: "perfetto"
+description:
+  "Perfetto system profiling and tracing"
+
 third_party {
+  identifier {
+    type: "ChromiumVersion"
+    value: "114.0.5735.338"  # from https://chromereleases.googleblog.com/2024/03/long-term-support-channel-update-for.html
```

Comment:
The blog URL shows 114.0.5735.355, not 114.0.5735.338.

Reply:
Yes but the tag exists for the newer commit too. I grabbed the latest tagged one from m114 release branch. Is there a reason why we'd want to pin to slightly older one ?

Reply:
> Yes but the tag exists for the newer commit too.

What tag? What newer commit?

> I grabbed the latest tagged one from m114 release branch. Is there a reason why we'd want to pin to slightly older one ?

To expand on what I wrote above, the blog post states:
“LTS-114 is being updated in the LTS channel to 114.0.5735.355”.
https://chromereleases.googleblog.com/2024/03/long-term-support-channel-update-for.html

Reply:
Nevermind - removing the URL as it has no significance. Updated the tag to `114.0.5735.356` which is latest available on `m114`

Reply:
The blog post is is significant in that it identifies a release
as mentioned more than one place in our documentation.
The tag `114.0.5735.356` does not correspond to a release.